static/_redirects

```diff
@@ -415,15 +415,29 @@
 /tutorials/Workspace_tutorial/ /tutorials/workspaces/ 301
 /tutorials/xgboost/ /tutorials/ 301
 /v/ / 301
-/wandb/config/ /ref/python/init/ 301
-/wandb/init/ /ref/python/init/ 301
-/wandb/log/ /ref/python/log/ 301
+/wandb/config/ /ref/python/sdk/functions/init/ 301
+/wandb/init/ /ref/python/sdk/functions/init/ 301
+/wandb/log/ /ref/python/sdk/log/ 301
 /weave/boards/ /guides/app/features/panels/query-panels/ 301
 /weave/streamtable/ /guides/app/features/panels/query-panels/ 301
 /weave/ /guides/app/features/panels/query-panels/ 301
 /weave/prod-mon/ /guides/core/ 301
 /zh/ / 301
 
+<!-- lazydocs migration -->
+/ref/python/data-types/graph/ /ref/python/data-types/ 301
+/ref/python/agent/ /ref/python/sdk/functions/agent/ 301
+/ref/python/artifact/ /ref/python/sdk/classes/Artifact/ 301
```

Comment:
This doesn't need to be cased and it could cause an issue. It's too bad that these can't easily be covered by a series of splats.